app/lib/populater/officers.rb

```diff
@@ -37,4 +39,17 @@ def self.populate_officer(officer)
 
     officer.save
   end
+
+  private
+  def self.populate_earnings_ranks
+    query = Officer.where.not(total: nil).order(total: :desc)
```

Comment:
I've always been writing stuff like `where("blah IS NOT NULL")` because I didn't know about this construct. I learned something today A+!